code/modules/antagonists/nukeop/nukeop.dm

```diff
@@ -87,6 +87,7 @@
 /datum/antagonist/nukeop/proc/memorize_code()
 	if(nuke_team && nuke_team.tracked_nuke && nuke_team.memorized_code)
 		antag_memory += "<B>[nuke_team.tracked_nuke] Code</B>: [nuke_team.memorized_code]<br>"
+		owner.add_memory(MEMORY_NUKECODE, list(DETAIL_NUKE_CODE = nuke_team.memorized_code, DETAIL_PROTAGONIST = owner.current), story_value = STORY_VALUE_SHIT, memory_flags = MEMORY_FLAG_NOLOCATION | MEMORY_FLAG_NOMOOD) 
```

Comment:
I'd say knowing the nuke codes would be pretty important and interesting, at least make it STORY_VALUE_NONE